src/test/java/org/graylog/integrations/notifications/types/SlackClientTest.java

```diff
@@ -54,7 +54,7 @@ public void send_sendsHttpRequestAsExpected_whenInputIsGood() throws Exception {
         assertThat(sent.url().toString()).isEqualTo("http://url.com/");
         assertThat(sent.method()).isEqualTo("POST");
         assertThat(sent.body()).isNotNull();
-        assertThat(sent.body().contentLength()).isEqualTo(76);
+        assertThat(sent.body().contentLength()).isEqualTo(76L);
```

Comment:
You can use `okio.Buffer` to check that the message sent in the request matches the message you expect (also, it might help to use a String constant or String variable rather than a String literal to make comparison and size checks easier).
```suggestion
        assertThat(sent.body().contentLength()).isEqualTo(Long.valueof(TEST_MESSAGE.length()));
        Buffer buffer = new Buffer();
        sent.body().writeTo(buffer);
        assertThat(buffer.readUtf8()).isEqualTo(TEST_MESSAGE);
```